client/src/main/java/io/pravega/client/state/SynchronizerConfig.java

```diff
@@ -21,6 +21,10 @@
 @Builder
 public class SynchronizerConfig implements Serializable {
 
+    /**
```

Comment:
Why is it that you added a comment here but not for the exact same field in the file below (`CorruptedStateException.java`) ?

Reply:
There are multiple such cases.

Reply:
Removed